src/app/components/PromotionModal/styled.ts

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
```

Comment:
We don't use `styled components` anymore. Rewrite it to `CSS modules` (see some `index.module.scss` for inspiration).